code/game/objects/items/weapons/manuals.dm

```diff
@@ -113,88 +113,44 @@
 
 
 /obj/item/book/manual/supermatter_engine
-	name = "Supermatter Engine User's Guide"
-	desc = "An engineer's best tool for dealing with their worst frenemy: The Supermatter."
+	name = "Supermatter Engine Anomaly Refrence"
+	desc = "An engineer's best tool for dealing with their worst frenemy: The Supermatter and its anomalous behavior."
 	icon_state = "bookParticleAccelerator"
-	author = "Waleed Asad"
-	title = "Supermatter Engine User's Guide"
+	author = "Vroo-Looum-Kloo"
+	title = "Supermatter Engine Anomaly Refrence"
 
 	pages = list({"Engineering notes on single-stage Supermatter engine,</br>
-			-Waleed Asad</br>
+			Vroo-Looum-Kloo</br>
 
-			A word of caution, do not enter the engine room, for any reason, without radiation protection and mesons on. The status of the engine may be unpredictable even when you believe it is .off.. This is an important level of personal protection.</br></br>
+			The supermatter engine is a very powerful, yet strange method of power generation. This guide will serve as a pocket refrence of the myraid of anomalous behaviors it may exhibit thoughout your shift. Below is a table of events and their effect.</br></br></br>
 
-			The engine has two basic modes of functionality. He has observed that it is capable of both a safe level of operation and a modified, high output mode.</br></br>
+			D Class: Events that only effect certain types of NON-STANDARD setups, minimial operator intervention required, they occur instantly and engineering will be alerted on telecomms.</br></br>
 
-			<center><b>Notes on starting the basic function mode, dubbed .Heat-Primary Mode..</b></center></br></br>
+			D-1: 200 mols of nitrous oxide are released by the crystal.</br>
+			D-2: 200 mols of nitrogen are released by the crystal</br>
+			D-3: 250 mols of CO2 are released by the crystal</br></br></br>
```

Comment:
```suggestion
			D-1: 200 moles of nitrous oxide are released by the crystal.</br>
			D-2: 200 moles of nitrogen are released by the crystal.</br>
			D-3: 250 moles of CO2 are released by the crystal.</br></br></br>
```

Reply:
For four years, I documented all my chem notes with mols. This didn't even occur to me lol.